hazelcast/src/main/java/com/hazelcast/sql/impl/schema/view/View.java

```diff
@@ -70,7 +64,6 @@ public boolean isStream() {
     public void writeData(ObjectDataOutput out) throws IOException {
         out.writeString(name);
         out.writeString(query);
-        out.writeBoolean(isStream);
```

Comment:
This must be backwards compatible, the class needs to implement `Versioned` and handle old data.